.github/scripts/bin_diff.ps1

```diff
@@ -166,18 +170,37 @@ function GetFilesWithHash {
 
 function Format-Color([hashtable] $Colors = @{}, [switch] $SimpleMatch) {
 	$lines = ($input | Out-String) -replace "`r", "" -split "`n"
+    $addedCount = 0
+    $deletedCount = 0
 	foreach($line in $lines) {
 		$color = ''
 		foreach($pattern in $Colors.Keys){
 			if(!$SimpleMatch -and $line -match $pattern) { $color = $Colors[$pattern] }
 			elseif ($SimpleMatch -and $line -like $pattern) { $color = $Colors[$pattern] }
 		}
 		if($color) {
-			Write-Host -ForegroundColor $color $line
+            if($color -eq 'Red'){
+                $deletedCount++
+			    Write-Host "`e[31;1m$line`e[0m"
+            }
+            if($color -eq 'Green'){
```

Comment:
I guess the color only works locally?

Reply:
This will work in Github Actions, I tested in one of the commits.